ui/app/AppLayouts/Profile/views/profile/MyProfileSettingsView.qml

```diff
@@ -0,0 +1,121 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.13
+
+import utils 1.0
+import shared 1.0
+import shared.panels 1.0
+import shared.popups 1.0
+import shared.controls.chat 1.0
+
+import "../../popups"
+import "../../stores"
+import "../../controls"
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+ColumnLayout {
+
+    readonly property bool dirty: displayNameInput.text != profileStore.displayName
+    readonly property bool valid: !!displayNameInput.text && displayNameInput.valid
+
+    function reset() {
+        displayNameInput.text = Qt.binding(() => { return profileStore.displayName })
+    }
+
+    function save() {
+        profileStore.setDisplayName(displayNameInput.text)
+    }
+
+    ProfileHeader {
+        Layout.fillWidth: true
+        Layout.leftMargin: Style.current.padding
+        Layout.rightMargin: Style.current.padding
+
+        displayName: profileStore.name
+        pubkey: profileStore.pubkey
+        icon: profileStore.icon
+        imageSize: ProfileHeader.ImageSize.Big
+        
+        displayNameVisible: false
+        pubkeyVisible: false
+        emojiHashVisible: false
+        editImageButtonVisible: true
+    }
+
+    StatusInput {
+        id: displayNameInput
+        Layout.fillWidth: true
+        label: qsTr("Display name")
+        input.placeholderText: qsTr("Display Name")
+        charLimit: 24
+        input.text: root.profileStore.displayName
+        validators: Constants.validators.displayName
+    }
+
+    StatusListItem {
+        Layout.fillWidth: true
+        title: qsTr("Biometric login and transaction authentication")
+        icon.name: "touch-id"
+        components: [ StatusSwitch {
+            id: biometricsSwitch
+            function toggleAccountSettings(storeToKeychainValue) {
+                localAccountSettings.storeToKeychainValue = storeToKeychainValue ? Constants.storeToKeychainValueStore : Constants.storeToKeychainValueNever;
```

Comment:
@igor-sirotin this line in case you're turning off biometrics should set local `storeToKeychainValue` to "not now", so you can be asked next time when you login whether you want to have pass stored to the keychain. But more important than that is... if you're turning on biometric, then you need to run "store to keychain process" so user may store his password to the keychain.

Reply:
@igor-sirotin an additional discussion may be opened with a topic whether biometric belongs to the page settings stuff or not, should we register this change and store it clicking on "Save changes" from the popup or just handle it independently. And the same question is for profile image, is it part of changes belonging under "Save changes" or independent change.